src/components/Navigation/Navigation.jsx

```diff
@@ -19,6 +19,7 @@ const Navigation = ({ setUserName }) => {
 };
 
 Navigation.propTypes = {
+  setUserName: PropTypes.func.isRequired
 };
```

Comment:
Weirdly, when I look at the react components in dev tools, it shows _no_ props for this component... but we are passing the `setUserName` prop _through_ this component. I'm not 100% sure if we need PropTypes in this file or not, but it doesn't seem to be saying anything is wrong, so maybe something to dig into another day for understanding.

Reply:
I would guess that's because it isn't being utilized in the file itself, but definitely worth learning about later. My impression based on the Turing lesson on PropTypes was that they are to validate the information being passed through to ensure that it is coming through in the format expected, so I think it may be redundant here without any mutation occurring but I'd rather leave it in for continuity along the data flow.

Reply:
Totally fair. Remind me to look into this further later!